memote/suite/cli/runner.py

```diff
@@ -676,12 +676,38 @@ def _setup_travis_ci(gh_repo_name, auth_token, repo_access_token):
         sys.exit(1)
 
     # Check if activation was successful
-    if t_repo["active"]:
-        LOGGER.info(
-            "Your repository is now on GitHub and automatic testing has "
-            "been enabled on Travis CI. Congrats!")
-    else:
-        LOGGER.critical("Unable to enable automatic testing on Travis CI!")
+    activated = False
+    for _ in range(60):
+        try:
+            LOGGER.info("Check if activation {} on Travis CI "
+                        "was successful".format(gh_repo_name))
+            response = requests.get(
+                "https://api.travis-ci.org/repo/{}".format(url_safe_repo_name),
+                headers=headers
+            )
+        except HTTPError as error:
+            LOGGER.critical(
+                "An error occurred. Please refer to the following error "
+                "code for further information:".format(str(error)))
```

Comment:
```suggestion
                "message for further information: {}".format(str(error)))
```

Reply:
I tried it out with the default settings and it works at least for me.